hazelcast-client/src/test/java/com/hazelcast/client/cache/nearcache/ClientCacheNearCacheBasicTest.java

```diff
@@ -149,10 +147,4 @@ protected ClientConfig getClientConfig() {
 
         return cacheConfig;
     }
-
-    @Test
-    @Override
-    @Ignore(value = "The NearCachedClientCacheProxy is missing `invalidateNearCache(keyData)` calls")
-    public void whenPutAllIsUsed_thenNearCacheShouldBeInvalidated_withUpdateOnNearCacheAdapter() {
```

Comment:
This test was just failing in the `CACHE_ON_UPDATE` configuration, which doesn't invalidate entries on the instance with the Near Cache. Ignoring this test actually skipped the wanted test for the `INVALIDATE` configuration as well. But with the separated tests we solved this issue.